openedx/features/course_experience/templates/course_experience/course-message-fragment.html

```diff
@@ -0,0 +1,32 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import get_language_bidi
+from openedx.core.djangolib.markup import HTML
+from openedx.features.course_experience import CourseHomeMessages
+%>
+
+<%
+is_rtl = get_language_bidi()
+%>
+
+<%block name="content">
+% if course_home_messages:
+    % for message in course_home_messages:
```

Comment:
I'm conflicted as to whether there should be a fragment per-message, or whether this should be a single fragment that renders a list of messages. I think it would be simpler for this to render a list of messages, but then I'd rename this file to be plural, plus make all the usages clearer about this too.